optimisation/index.md

```diff
@@ -1,7 +1,6 @@
 # Module content
 
-- ToDo
-- ToDo
-- ToDo
-- ToDo
-- ToDo
+1. Intro (how does RN works, why performance issues can be faced)
+2. How to measure performance of RN app
+3. Common ways to improve performance (console statements; FlatList; React.memo(); useMemo()/useCallback(); images: sizes and format, caching; aminations; unnecessary dependencies; Reselect; inline anonymous functions
```

Comment:
How about adding Fast refresh? It could increase the coding performance
https://reactnative.dev/docs/fast-refresh

Reply:
I don't see how Fast Refresh is connected to app's optimisation. It's just a development tool